2D3D_UnityProject/Assets/Scripts/Puzzles/Zodiac/ZodiacPuzzle.cs

```diff
@@ -4,6 +4,11 @@
 
 public class ZodiacPuzzle : MonoBehaviour
 {
+    /// <summary>
+    /// The number of rounds that must be solved in order to complete the puzzle
+    /// </summary>
+    public const int numberOfRounds = 3;
```

Comment:
yeah here (see prev. comment), we should check to make sure each zodiac disk has this many correct symbols (in Start() or somethin)